src/monstergenerator.cpp

```diff
@@ -547,8 +548,10 @@ void mtype::load( JsonObject &jo, const std::string &src )
     }
 
     assign( jo, "color", color );
-    const typed_flag_reader<decltype( Creature::size_map )> size_reader{ Creature::size_map, "invalid creature size" };
-    optional( jo, was_loaded, "size", size, size_reader, MS_MEDIUM );
+    //const typed_flag_reader<decltype( Creature::size_map )> size_reader{ Creature::size_map, "invalid creature size" };
```

Comment:
Please do not check in commented out code.

Reply:
good call, didn't even noticed that I left it behind